packages/logout-extension/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin,
+  IRouter,
+} from '@jupyterlab/application';
+
+import '@jupyterlab/application/style/buttons.css';
+
+import '../style/index.css';
+
+const logoutPluginId = '@jupyterlab/logout-extension:plugin';
```

Comment:
Since this is not an official extension, maybe the following name would be more appropriate?

```suggestion
const logoutPluginId = 'jupyterlab-logout-extension:plugin';
```

Reply:
For consistency we could then do the same with the other extensions in this repo / diff.

Reply:
Fair enough. I thought this convention was put in-place when extensions were distributed as npm packages to put all official extensions under one umbrella namespace. As this is not the case anymore, I thought we could use this convention so that all extensions will be installed at same location for the end users. 

Anyways, I will change the names for all extensions.